nni/contrib/compression/utils/evaluator.py

```diff
@@ -418,9 +464,40 @@ def unbind_model(self):
         else:
             _logger.warning('Did not bind any model, no need to unbind model.')
 
-    def _patch_configure_optimizers(self):
+    def patch_optim_param_group(self, module_name_param_dict: Dict[str, List[Tensor]] | None = None):
```

Comment:
why `module_name_param_dict` can be None?

Reply:
done